### PR TITLE
PXC-3286 - Fixed Galera/PXB24 build issues 

### PR DESCRIPTION
**Pull request by rameshvs02**

### CODE REVIEW

File: pxc/jenkins/pxc57-pipeline.groovy [hunk 1]
```diff
@@ -99,50 +89,12 @@ pipeline {
...
-                                fi
-                            '''
-                        }
-                    }
-                }
                 stage('Build PXB24') {
                     agent { label 'docker' }
                     steps {
-                        git branch: 'master', url: 'https://github.com/Percona-Lab/jenkins-pipelines'
+                        git branch: 'PXC-3286-updates', url: 'https://github.com/Percona-Lab/jenkins-pipelines'
```
Comment (hors) [inline pxc/jenkins/pxc57-pipeline.groovy:97]: 'master' branch

File: pxc/jenkins/pxc57-pipeline.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@
         - git:
             url: https://github.com/Percona-Lab/jenkins-pipelines.git
             branches:
-            - 'master'
+            - 'PXC-3286-updates'
```
Comment (hors) [inline pxc/jenkins/pxc57-pipeline.yml:12]: please fix the branch name in all files

File: pxc/jenkins/pxc57-pipeline.groovy (hunk 1 above)
Comment (rameshvs02) [reply pxc/jenkins/pxc57-pipeline.groovy]: Fixed

File: pxc/jenkins/pxc57-pipeline.yml (hunk 2 above)
Comment (rameshvs02) [reply pxc/jenkins/pxc57-pipeline.yml]: Fixed

**Pull request merged**